uml/capsule_update_tech_flow.puml

```diff
@@ -0,0 +1,88 @@
+@startuml
+!pragma teoz true
+
+skinparam shadowing<<with_shadow>> true
+skinparam NoteTextAlignment<<with_shadow>> center
+skinparam NoteBackgroundColor<<with_shadow>> #CEF
+
+participant "UEFI Shell" as OS
+activate OS
+participant coreboot
+box UEFI
+participant PEI
+participant DXE
+collections Variables as vars
+end box
+collections RAM as data order 50
+
+hnote over data<<with_shadow>>
+Represents data in RAM,
+such as UEFI HOBs, coreboot
+CBMEM and system tables
+end hnote
+
+opt
+note over OS: Run CapsuleApp.efi
+OS <-- Header: Load and validate Capsule image
+data <-- OS: Store (possibly scattered) Capsule image in RAM
+vars <-- OS: Save information about Capsules
+end opt
+
+OS -> coreboot --++: Reboot
+
+coreboot <-- vars: Check for Capsule presence
+
+opt
+coreboot <-- data: Read scattered Capsules from RAM
+note over coreboot: Coalesce\nCapsule(s)
```

Comment:
It's "Capsule(s)" here but "Capsule" or "Capsules" in other places.  I guess it can just be plural for consistency.

Reply:
https://github.com/Dasharo/docs/pull/855/commits/70a5dfc728aa75cd04e4f50c781e1853e71fa60a#diff-064833a8bac51a7baefcd5053812ef902a9960ef2803764c0a329ace5aa05898R37